### DISCUSSION TITLE
How to disable tooltip and legend?

### DISCUSSION THREAD

**Discussion by paulorb** [Live-Charts/LiveCharts2#357] [Q&A] [answered]
Hi,

First of all nice project,  very easy to use and has a lot of potential.  I am trying to disable the tooltip, I reviewed the documentation and now I am looking at the source code. I tried to change the constructor of the CartesianChart since it receives 
IChartTooltip and IChartLegend. But it doesn't disable it, the tooltip is still there.  Is it enabled by default?

`this.cartesianChart = new LiveChartsCore.SkiaSharpView.WinForms.CartesianChart(null,null);`

![image](https://user-images.githubusercontent.com/2589346/148353672-d9b4c27b-4da8-4ff6-94fb-6723d3abecd9.png)

Thanks,
Paulo

**Comment by paulorb** [comment, marked as answer]
I found it

cartesianChart.TooltipPosition = TooltipPosition.Hidden;

**Reply by lorenzua02** [reply]
thanks! the XAML equivalent is
```cs
<lvc:CartesianChart Series="{Binding Series}" TooltipPosition="Hidden">
```
